.github/dependabot.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    groups:
+      github-action-dependencies:
+        patterns:
+          - "*"
+
+  - package-ecosystem: "npm"
+    directory: "/M2/Macaulay2/editors/prism"
```

Comment:
Would it make more sense to also move every file other than `macaulay2.js.in` and perhaps `README.md` to Style? Then prism and katex can update together and the build target in this directory will generate the grammar file and copy it into the Style package.

Reply:
Yeah, I though about that too.  I'll play around with it.